### PR TITLE
Fix pom.xml for liberty profile.

### PR DESCRIPTION
**Pull request by daggerok**

### CODE REVIEW

**Comment by daggerok** [comment]
whats is wrong with MRs for eclipse and agreements? I already signed it

**Comment by rdebusscher** [comment]
Hi @daggerok 

You need to sign your commits, see https://github.com/eclipse/microprofile-starter/blob/master/CONTRIBUTING.md

File: src/main/resources/pom-servers.xml [hunk 1]
```diff
@@ -151,7 +151,7 @@
                             </execution>
                         </executions>
                         <configuration>   
-                            <appArchive>${project.build.directory}/${final.name}.war</appArchive>
+                            <archive>${project.build.directory}/${final.name}.war</appArchive>
```
Comment (rdebusscher) [inline src/main/resources/pom-servers.xml]: Closing tag is not the same as opening tag.

**Comment by rdebusscher** [comment]
@daggerok Can you open an issue indicating why the pom.xml needs to be changed? (as the current one works fine)

**Comment by daggerok** [comment]
It's easy to reproduce with for example idea IDE:

1) generate open-liberty mp 3 project on https://start.microprofile.io/
image

2) unzip demo.zip and import it in idea IDE as a maven project and see validation error
image

3) by using code completion for openliberty plugin I can see correct configuration property:
image

4) finally by using command+b hotkey I can deep dive into openliberty maven plugin xsd schema definition to read generated from javaDoc description, so here you can clearly see that appArchive property doesn't exists

PS: It's hard to understand how and what I should do to easily create and issue.

---
Regards,
Maksim

**Comment by Karm** [comment]
@daggerok Hello,

The "signing part" is actually just a text in your commit message. It is not required to have the commits cryptographically signed. See this guide I wrote a while back, screenshots included:

https://github.com/eclipse/microprofile-starter/pull/108#issuecomment-497310175

File: src/main/resources/pom-servers.xml (hunk 1 above)
Comment (Emily-Jiang) [reply src/main/resources/pom-servers.xml]: I suggest to delete this line altogether as it is not needed.

**Comment by Emily-Jiang** [comment]
@daggerok thanks for reporting this! You need to sign off ECA and then use `commit -s` to sign off your commit. By the way, I suggest to delete the line in question as it is not needed.

**Comment by Emily-Jiang** [comment]
@daggerok are you going to delete the line as per my comment https://github.com/eclipse/microprofile-starter/pull/242#discussion_r351864255

**Comment by daggerok** [comment]
Hello @Emily-Jiang ,
I thought it would be better to use proper configuration, but okay, removing this...

**Pull request merged**